### ISSUE TITLE
Need more tuning of mmod_face_detector.dat's parameters

### ISSUE THREAD

**Issue by KevinPatel04** [davisking/dlib-models#33]
Dear @davisking ,
My name is Kevin Patel, I am 3rd Year B.Tech student in Computer Science program. I and my team are building a system for video-base dynamic human face recognition as a part of my mid term project. We  have used pre-trained CNN model (MMOD) which is fast and accurate than other models (like MTCNN which is accurate but very slow for live applications) that we have tried, but the problem with this is that this model can detect the human faces with good accuracy up to some threshold distance only. If the person is standing away from that threshold than his/her face is not detected. And moreover we are trying to detect and recognize face irrespective of angle, distance & amount of light. So for producing better optimized results, we are trying to tune the hyper parameters. So on searching on web, we found your paper on Max Margin Object Detection (MMOD) published on 2015, in which you have mentioned MMOD Optimizer (Algorithm 2). But we don't know from where to start from can you guide us for the same. Sir can you provide python code or more detailed algorithm for parameters tuning which we can use. Sir for your reference I have attached the pre-trained model file mmod_human_face_detector.dat that we are currently using.

It would be very helpful for us if you look and help us with this parameters tuning, waiting for positive reply from your side.
